sap-successfactors/4.0/modules/ROOT/pages/sap-successfactors-connector-examples.adoc

```diff
@@ -1,161 +1,538 @@
-
-== <Example 1>
-
-// Use subsections, if needed, to show configuration steps. 
-=== Subsection 1 
-=== Subsection 2
-=== Subsection 3
-.
-.
-.
-// Add these steps to the end of the numbered list:
-. Save the project. 
-. Test the app by sending a <REST command> request to <url>.
-
-=== XML for <what the example shows>
-
-Paste this code into the Studio XML editor to quickly load the flow for this example into your Mule app:
-
-// <Copy the XML from Studio and paste it here. 
-// Remove the  attributes from the XML before including it in the topic.>
-
-// If you aren’t splitting the example into subsections, add these subsections instead:
-
-=== Steps for <what the example shows>
-
-// Add these steps to the end of the numbered list:
-. Save the project. 
-. Test the app by sending a <REST command> request to <url>.
-
-=== XML for <what the example shows>
-
-Paste this code into your Studio XML editor to quickly load the flow for this example into your Mule app:
-
-// <Copy the XML from Studio and paste it here. 
-// Remove the  attributes from the XML before including it in the topic.>
-
-== <Example 2>
-
-// Same template for Example 1
-
-== <Example 3>
-.
-.
-.
+These examples show how to create flows to invoke some of SuccessFactor's standard operations, including how to:
+
+* <<create-query-flow,Create a query to query across entities>>
+* <<create-entry,Create an entry on SuccessFactors>>
+* <<get-entity-by-id,Get an entity by ID>>
+* <<update-entry,Create and update an entry on SuccessFactors>>
+* <<delete-entry, Delete an entry from SuccessFactors>>
+
+== Prerequisites
+
+To build and run these flows, you need:
+
+* OpenJDK 8 or 11
+* Anypoint Studio 7.1 or later
+* Mule 4.2.2 or later
+* SuccessFactors Connector 3.x or later
+* SuccessFactors credentials to send OData API requests
+* DataWeave 2.0
+
+=== Prerequisite Steps
+
+. <<create-mule-project,Create a Mule Project>>
+. <<specify-credentials,Specify Your Credentials>>
+. <<configure-global-element,Configure a Global Element>>
+
+[[create-mule-project]]
+=== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector: 
+
+. In Studio, select *File > New > Mule Project*.
+. In *Project Name*, enter a name for your Mule project. +
+This example uses the name `successfactors-operations`. 
+. Click *Finish*.
+
+=== Specify Your Credentials
+
+. In *Package Explorer*, under `/src/main/resources/`, create a file named `mule-app.properties`. 
+. Specify your credentials for connecting to SuccessFactors, for example:
++
+----
+config.companyId=<your-company-id>
+config.userName=<successfactors-username>
+config.password=<successfactors-password>
+----
++ 
+. Save the file. 
+
+[[configure-global-element]]
+=== Configure a Global Element for the Connector
+
+Configure a global element to provide the authentication credentials that the connector requires to access the target SAP SuccessFactors system.
```

Comment:
You have already mentioned SuccessFactors, so the SAP qualifier should go on first mention and not here.